### ISSUE TITLE
SwinUNETR Pretraining ignores in_channels != 1 in reconstruction loss

### ISSUE THREAD

**Issue by ardapekis** [Project-MONAI/research-contributions#82]
**Describe the bug**
The SwinUNETR pretraining script accepts in_channels greater than 1, but the reconstruction loss calculation in this case is wrong since the model only outputs 1 channel, regardless of the given number.

**To Reproduce**
1. Construct a multichannel dataset compatible with SwinUNETR pretraining.
2. Run SwinUNTER pretraining with `--in_channels N` where N > 1.

**Expected behavior**
The output channels of the reconstruction head of the SwinUNETR should match the input channels.

**Screenshots**
![image](https://user-images.githubusercontent.com/10730618/179607918-addf2be1-f396-45b3-891a-62c3574776c0.png)

**Environment (please complete the following information):**
 - OS: `amazon/Deep Learning AMI GPU PyTorch 1.12.0 (Ubuntu 20.04) 20220706`
 - Python version: `3.9.13`
 - MONAI version [e.g. git commit hash]: `0.9.1rc3+4.g5fb767c`
 - CUDA/cuDNN version: 11.6
 - GPU models and configuration: 1 x Tesla T4

**Comment by wyli** [comment]
I think the code is only tested with the commit listed here https://github.com/Project-MONAI/research-contributions/blob/c6e7ac8c61469736e6df394b7050ef25507c904b/SwinUNETR/Pretrain/requirements.txt#L1 could you please try that version as well? (though I'm very curious what's not working with the latest dev.)

**Comment by ardapekis** [comment]
Strangely I don't seem to be able to install that particular version of MONAI. It is resolving to `monai-0.9.1rc3+4.g5fb767c` even if I uninstall and explicitly specify it.

This issue is not related to MONAI though, it is a simple hardcoded number that seemed to be forgotten.

**Comment by wyli** [comment]
thanks, I've updated the requirements.txt files https://github.com/Project-MONAI/research-contributions/pull/84/files
